apps/console/src/extensions/configs/application.tsx

```diff
@@ -83,26 +78,35 @@ export const applicationConfig: ApplicationConfig = {
         getOverriddenImage: (_clientId: string, _tenantDomain: string) => {
             return null;
         },
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Remove suppression. Add `_` to the front of the var instead.
```suggestion
getTabPanelReadOnlyStatus: (_tabPanelName: string, _applicationName: ApplicationInterface)
```

Reply:
Fixed with b162ba9502b5fc4e15d504f3f3ae0ae36f94a755